CMSgov/mint-app
# [EASI-2212] Front end for Read Only Model Summary

# EASI-2212

## Changes and Description

- Adds a read-only Model Plan Summary
- Adds a new `GetModelSummary` query and types
- Update `TaskListStatus` to take in a few more props 



## How to test this change

1. Create a new model plan and fill out the following items:
    1. Model Name
    2. Model Basics -> Goal 
![image](https://user-images.githubusercontent.com/2349702/184945875-eeeb290c-0a72-49ec-8b00-13e9a18d530f.png)

    3. Model Basics -> Application start date 
![image](https://user-images.githubusercontent.com/2349702/184945925-0e8942d2-44f0-4372-9c66-b80b75373c99.png)

    4. General Characteristics -> Key Characteristics 
![image](https://user-images.githubusercontent.com/2349702/184945976-28f62e53-cb69-4151-abed-632cbee1bb87.png)

2. Click `View this Model Plan` in side nav 
![image](https://user-images.githubusercontent.com/2349702/186009477-6ffa6960-69ed-4047-be6b-ec1c829af7f3.png)

3. Ensure all fields correctly show up 
![image](https://user-images.githubusercontent.com/2349702/184946028-26699895-dbfa-4c0f-812f-860e7b544753.png)
4. Click on `Update this Model Plan`
5. Change the status to either `Cleared` or `Announced` 
6. Click `Update` and page should return to the Summary, not task list
7. Ensure Status tag updates and that the alert is not showing







## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
